### ISSUE TITLE
Never use a URL shortening service - even if you own it

### ISSUE THREAD

**Issue by nelsonic** [dwyl/linky#17]
https://news.ycombinator.com/item?id=34858409

I consider this to be good feedback. That we need to incorporate into our plan/roadmap.

**Comment by iteles** [comment]
A perspective I hadn't considered. 👌🏻

**Comment by nelsonic** [comment]
Yeah, but the `Guardian` is being **_really_ short-sighted** about the sale of `gu.com` 🤦‍♂️ 
They had **`Revenue £255.8m`** and **Operating Profit: `£11.7m`** in 2022.
See: https://www.theguardian.com/gnm-press-office/2022/jul/20/guardian-media-group-plc-gmg-publishes-202122-statutory-financial-results
`if` someone `else` has the `£2.5M` to buy the domain they can invalidate all the existing links (as the tweet suggests).
This is a classic bean-counter move. 
Just because they cannot figure out how to "monetize" it. 
It's the same BS Microsoft pulled when they bought `GitHub` and discontinued `git.io`:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/194400/223565856-f9f1fe1f-0e76-49c0-b506-ef9f8212a3e5.png">

I _wonder_ who will end up "buying" (taking over) the domain. 💭
